Base/etc/LookupServer.ini

```diff
@@ -1,2 +1,2 @@
 [DNS]
-Nameserver=1.1.1.1
+Nameservers=1.1.1.1,1.0.0.1
```

Comment:
Off-topic: Shouldn't these info be supplied by DHCP?

Reply:
Not sure what the current state of DHCP in Serenity is, but yeah, eventually it should do that :)

Reply:
Yeah that would be a cool thing to add to the `DHCPClient` service :)